client/ayon_blender/api/ops.py

```diff
@@ -41,23 +41,31 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-class BlenderApplication(QtWidgets.QApplication):
+class BlenderApplication:
     _instance = None
     blender_windows = {}
 
-    def __init__(self, *args, **kwargs):
-        super(BlenderApplication, self).__init__(*args, **kwargs)
-        self.setQuitOnLastWindowClosed(False)
-
-        self.setStyleSheet(style.load_stylesheet())
-        self.lastWindowClosed.connect(self.__class__.reset)
-
     @classmethod
     def get_app(cls):
         if cls._instance is None:
-            cls._instance = cls(sys.argv)
+            # If any other addon or plug-in may have initialed a Qt application
+            # before AYON then we should take the existing instance instead.
+            application = QtWidgets.QApplication.instance()
+            if application is None:
+                application = QtWidgets.QApplication(sys.argv)
+
+            # Ensure it is configured to our needs
+            cls._prepare_qapplication(application)
+            cls._instance = application
+
         return cls._instance
 
+    @classmethod
+    def _prepare_qapplication(cls, application: QtWidgets.QApplication):
+        application.setQuitOnLastWindowClosed(False)
+        application.setStyleSheet(style.load_stylesheet())
```

Comment:
If there is existing application, would this possible break their styles?

Reply:
Yes - we're basically overriding the `QApplication` style, so any other widgets by any other Qt tool would also inherit those styles from the QApplication. Yes.

Reply:
Because I do expect that the error happened because other addon created QApplication, so we might break styles of other addon (or other addon can break our styles).

I'm curious if we need to set the styles for whole application?

Reply:
Yup - I'm also curious as to why that logic is there. I believe all widgets we're opening get their stylesheets applied in a dedicated manner anyway so we should be able to rely on that.

Reply:
Maybe there are popup dialogs that don't set it?